src/style.css

```diff
@@ -1 +1,53 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(
+  );
+}
+
+body {
+  font-family: Roboto, serif;
+  font-size: 12px;
+  margin: 0;
+  max-width: 1200px;
+  height: 60px;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
```

Comment:
You should add left and right paddings for header 50 px instead of using margins for logo and navigation

Reply:
Use correct paddings and justify-content: space between for positioning elements